src/vs/workbench/contrib/chat/browser/media/chatSetup.css

```diff
@@ -23,6 +27,14 @@
 		font-size: 12px;
 		color: var(--vscode-descriptionForeground);
 	}
+
+	.chat-setup-dialog-icon-background {
+		position: absolute;
+		right: -50px;
+		top: -20px;
+		font-size: 240px !important;
```

Comment:
This wasn't a mistake? That's 240px on my PC:

![image](https://github.com/user-attachments/assets/55e06cf6-7099-4d49-a362-c16d17e44957)

Reply:
oh it's for the codicon 👌